### PR TITLE
mapping file fixes

### PR DESCRIPTION
**Pull request by squirrelo**
This is a collection of all fixes needed before updates to EBI

### CODE REVIEW

**Comment by mortonjt** [comment]
Are the tests supposed to fail?

File: knimin/lib/util.py [hunk 1]
```diff
@@ -198,6 +198,26 @@ def categorize_age(x):  # noqa
     return age_cat
 
 
+def correct_age(age, height, weight, etoh):
+    """Infers incorrect ages and incorrectly classified babies"""
+    # Checks the logic for age
+    if age >= 4 and age < 123:
```
Comment (mortonjt) [inline knimin/lib/util.py]: Why can't we consider ages between 0 and 4?

Comment (squirrelo) [inline knimin/lib/util.py]: those are the ages we are making sure are correct through the height, weight, and alcohol checks.

Comment (mortonjt) [inline knimin/lib/util.py]: Alright! Makes sense.

Could you rename the function to something like `fix_babies_age` or something?

**Comment by mortonjt** [comment]
One minor comment, then I think it is ready to merge.

File: knimin/lib/util.py (hunk 1 above)
Comment (squirrelo) [inline knimin/lib/util.py]: it's not just babies though, it's also highlanders (people over 123 years old) and time travelers (people with negative ages)

Comment (mortonjt) [inline knimin/lib/util.py]: ahhh :L 

ok cool.  I think this is good

**Pull request merged**